packages/integration-tests/test/standardExitTest.js

```diff
@@ -349,18 +349,6 @@ describe('standardExitTest.js', function () {
       // Check utxos on the child chain
       const utxos = await childChain.getUtxos(aliceAccount.address)
       assert.equal(utxos.length, 1)
-      assert.hasAllKeys(utxos[0], [
```

Comment:
`assert.containsAllKeys()`? and same for other tests?

Reply:
yea thought it would be better to not have to "maintain" this assert, so the test will fail further down the line if the necessary keys dont exit